pkg/storage/secret/encryption/data/encrypted_value_list_all.sql

```diff
@@ -0,0 +1,17 @@
+SELECT
+  {{ .Ident "namespace" }},
+  {{ .Ident "name" }},
+  {{ .Ident "version" }},
+  {{ .Ident "encrypted_data" }},
+  {{ .Ident "created" }},
+  {{ .Ident "updated" }}
+FROM
+  {{ .Ident "secret_encrypted_value" }}
+{{ if .HasUntilTime }}
```

Comment:
Would it make sense to remove the `.HasUntilTime` and use the `.UntilTime` variable itself to check, or would there be a valid case for `created <= 0`?
```suggestion
{{ if (gt .UntilTime 0) }}
```